rpcd/playbooks/roles/rpc_support/tasks/retrieve_auth_key.yml

```diff
@@ -13,6 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: Install pip packages for memecache module to work
```

Comment:
I wasn't able to find the exact commit that causes these packages to no longer be installed. Rather I just noticed that they no longer existed.